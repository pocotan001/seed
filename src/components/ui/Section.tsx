import margin, { IMarginProps } from "~/components/styles/extends/margin";
import styled from "~/components/styles/themedStyledComponents";

interface ISectionProps extends IMarginProps {
  children: React.ReactNode;
}

const Section = styled<ISectionProps, "section">("section")`
  ${margin};
`;

export default Section;
