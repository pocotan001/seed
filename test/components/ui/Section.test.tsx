import * as React from "react";
import * as renderer from "react-test-renderer";
import Section from "~/components/ui/Section";

describe("<Section>", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Section>alo</Section>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
