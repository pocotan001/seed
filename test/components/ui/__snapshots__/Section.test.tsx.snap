// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Section> should render correctly 1`] = `
<section
  className="Section-sa1b69-0 lfoNbn"
>
  alo
</section>
`;
